include/inviwo/core/datastructures/image/layerram.h

```diff
@@ -184,4 +184,15 @@ auto LayerRAM::dispatch(Callable&& callable, Args&&... args) const -> Result {
                                                     std::forward<Args>(args)...);
 }
 
+class IVW_CORE_API LayerRamResizer {
+public:
+    virtual ~LayerRamResizer() = default;
+
+    /**
+     * Resize and copy src to dst doing some kind of content scaling
+     * @see CIMGLayerRamResizer
+     */
+    virtual bool resize(const LayerRAM& src, LayerRAM& dst) const = 0;
```

Comment:
```suggestion
     /**
     * Resample the contents of src to dst while taking their potentially different sizes into account.
     * @see CIMGLayerRamResizer
     */
    virtual bool resample(const LayerRAM& src, LayerRAM& dst) const = 0;
```

Reply:
// copy the data from \p src to \p dst. This might involve up or down scaling if the dimensions do not match.
// The dimensions of both \p src and \p dst will not change.